### PR TITLE
[r3-corda-ent] enable platform deployment via ansible-server

### PR DESCRIPTION
**Pull request by saikumarbommakanti**
feat (r3-corda-ent): enable platform deployment via ansible-server

This commit introduces support for deploying a decentralized ledger technology (DLT) network using Ansible automation. 

- Updated the Ansible codebase to support network deployment in respect of the standalone Helm chart
- The following Ansible roles have been introduced to appropriately deploy the network of one organization only: 1. initlize the organization 2. setup cenm 3. setup notary 4. setup networkmap

- updated the sample network configuration file to provide information on which networks can be deployed using this file 

fixes hyperledger#2548

### CODE REVIEW

File: platforms/network-schema.json [hunk 1]
```diff
@@ -970,7 +970,7 @@
       "type": "object",
       "properties": {
         "keystore": {
-          "type": "object",
```
Comment (suvajit-sarkar) [inline platforms/network-schema.json:973]: kindly review the schema, for the credentials. doesn't match with current network.yaml structure

File: platforms/r3-corda-ent/charts/cenm-networkmap/templates/service.yaml [hunk 2]
```diff
@@ -47,7 +47,8 @@ kind: Host
 metadata:
   name: {{ .Release.Name }}-nms
 spec:
-  hostname: cenm-nms.{{ .Values.global.proxy.externalUrlSuffix }}
+  # hostname: cenm-nms.{{ .Values.global.proxy.externalUrlSuffix }}
```
Comment (suvajit-sarkar) [inline platforms/r3-corda-ent/charts/cenm-networkmap/templates/service.yaml:50]: remove the code comments

File: platforms/r3-corda-ent/charts/cenm-networkmap/templates/service.yaml [hunk 3]
```diff
@@ -47,7 +47,8 @@ kind: Host
...
+  hostname: {{ .Values.global.proxy.prefix }}-nms.{{ .Values.global.proxy.externalUrlSuffix }}
```
Comment (suvajit-sarkar) [inline platforms/r3-corda-ent/charts/cenm-networkmap/templates/service.yaml:51]: prefix which should match the cenm release name should be non global value

**Review by suvajit-sarkar — CHANGES_REQUESTED** [review]

**Comment by suvajit-sarkar** [comment]
The PR does not cover deploying organization with node. Kindly create an issue.

File: platforms/r3-corda-ent/charts/cenm-networkmap/templates/service.yaml (hunk 3 above)
Comment (saikumarbommakanti) [reply platforms/r3-corda-ent/charts/cenm-networkmap/templates/service.yaml:50]: Its a non global value as it is coming from networkmap.tpl (  .Values.global.proxy.prefix )

**Review by suvajit-sarkar — APPROVED** [review]

**Pull request merged**